omc3/model/accelerators/skekb.py

```diff
@@ -21,5 +24,37 @@ def verify_object(self):  # TODO: Maybe more checks?
             raise AcceleratorDefinitionError("SuperKEKB doesn't have a model creation, calling it this "
                                              "way is most probably wrong.")
 
-    def get_beam_direction(self):
+    @staticmethod
+    def get_class_parameters():
+        params = EntryPointParameters()
+        params.add_parameter(flags=["--ring"], help="HER or LER ring.", name="ring", type=str)
```

Comment:
I would add choices=("ler", "her") , this will also clarify the condition in line 44